### ISSUE TITLE
Do not understand the "Run On" section of Keypress

### ISSUE THREAD

**Issue by TheJessaChannel** [rivafarabi/deckboard#166]
I want to press ctrl+alt+numpad9 to save a replay in OBS. The keybind works just fine in OBS, but cannot figure out how to set this up in Deckboard.

By the way, this is a fantastic app, bought the pro version, well worth it! Thank you!

**Comment by GalacticZero** [comment]
Yeah, that is one thing I'm having trouble with. I want to perform a set of keypresses on an application, but just creating that in the program doesn't make it do that. And "Run On" is empty. Isn't that what it is supposed to be for? Telling it which window to send the keypresses to?
